src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+                break;
+            }
+
+            //getting index for mark and delete
+            String[] find_index = line.split(" ");
+
+            //Task description
+            String desc;
+            desc = line;
+            Task t = new Task(desc);
+
+            if (line.toLowerCase().contains("unmark") || (line.toLowerCase().contains("mark")) || line.toLowerCase().contains("delete"))
+            {
+                if (line.toLowerCase().contains("unmark")) {
+                    index_for_mark = Integer.parseInt(find_index[1]);
+                    tasks.get(index_for_mark - 1).markAsUnDone();
+
+                    System.out.println("____________________________________________________________\n"
+                            + "OK, I've marked this task as not done yet:\n"
+                            + tasks.get(index_for_mark - 1) + "\n"
+                            + "____________________________________________________________\n");
+
+
+                } else if (line.toLowerCase().contains("mark")) {
+                    index_for_mark = Integer.parseInt(find_index[1]);
+                    tasks.get(index_for_mark - 1).markAsDone();
+                    System.out.println("____________________________________________________________\n"
+                            + "Nice! I've marked this task as done:\n"
+                            + tasks.get(index_for_mark - 1) + "\n"
+                            + "____________________________________________________________\n");
+
+
+                } else if (line.toLowerCase().contains("delete")) {
+                    int index_for_delete = Integer.parseInt(find_index[1]);
+                    System.out.println("____________________________________________________________\n"
+                            + " Noted. I've removed this task:\n" +
+                            "  " + tasks.get(index_for_delete - 1) + "\n" +
+                            "Now you have " + (i - 1) + " tasks in the list.\n"
+                            + "____________________________________________________________\n");
+                    i -= 1;
+                    tasks.remove(tasks.get(index_for_delete - 1));
+
+                }
+                String newString = "";
+                String filePath = "Duke.txt";
+
+                for (int m = 0; m < i; m += 1) {
+                    int index = m + 1;
+                    newString += (tasks.get(m) + "\n");
+                }
+
+
+                try {
+                    writeToFile("Duke.txt", newString);
+                } catch (IOException e) {
+                    System.out.println("Something went wrong\n"
+                            + "____________________________________________________________");
+                }
+            }
+
+            else if (line.toLowerCase().contains("todo") || line.toLowerCase().contains("deadline") || line.toLowerCase().contains("event")) {
+
+                boolean empty;
```

Comment:
Consider renaming this variable to something like `isEmpty`. To quote the coding standards "Boolean variables/methods should be named to sound like booleans." [See here](https://se-education.org/guides/conventions/java/basic.html#naming)